style/rails/migration.rb

```diff
@@ -1,9 +1,10 @@
 class CreateClearanceUsers < ActiveRecord::Migration
   def change
     create_table :users  do |t|
-      t.timestamps null: false
       t.string :email, null: false
       t.string :name, null: false, default: ''
+
+      t.timestamps null: false
```

Comment:
I think that would be fine, too. So, we'd leave the example as it was before this pull request, and remove the guideline entirely.